jobs/haproxy/spec

```diff
@@ -73,9 +73,11 @@ properties:
     default: false
   ha_proxy.disable_domain_fronting:
     description: |
+      Must be one of "true", "false", or "mtls_only"
```

Comment:
Since this is no longer a boolean value, I think the default should now be "false" instead of false. Not sure if Ruby likes mixing the type.

Reply:
Ruby doesn't care. But I'll add "false" and "true" (strings) as undocumented options since the intent of the user is clear in these cases